proposals/shared-everything-threads/Overview.md

```diff
@@ -86,8 +86,9 @@ We expect the following changes to enable something like the following example (
 
   ;; $bar cannot access the un-shared memory, but it can call the shared import $baz.
   (func $bar (type $shared)
-    call $baz
+    call $baz ;; validation error!
```

Comment:
This is not a validation error because both `$bar` and `$baz` are shared, right?

Reply:
Yup, wasn't looking closely enough.